internal/observability/event/context.go

```diff
@@ -213,7 +216,7 @@ func addCtxOptions(ctx context.Context, opt ...Option) ([]Option, error) {
 			// there's no RequestInfo, so there's no id associated with the
 			// observation and we'll generate one and flush the observation
 			// since there will never be another with the same id
-			id, err := newId(string(ObservationType))
+			id, err := NewId("e")
```

Comment:
It's not like you to not use string constants. Curious about this change.

Reply:
It's not just "observations" any longer and I need to update the comment. ty!

I'll add a const as well.